documentation/module_structure.md

```diff
@@ -10,6 +10,12 @@ root of the project directory and contains a `name` key, the project itself is
 also loaded as a module and namespaced to the value of `name`. For more
 information on `bolt-project.yaml`, see [Bolt projects](projects.md).
 
+## 🧪 New experimental module management
```

Comment:
I don't think the new module management workflow makes any difference to the module structure - to me this link feels a little confusing to readers who come to this page, but I don't feel strongly about it.